front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -149,15 +221,34 @@
         display: flex;
         align-items: center;
         justify-content: space-between;
-        gap: 0.5rem;
-        margin-bottom: 0.5rem;
-        height: 2rem;
-
+        height: 2.375rem;
+        margin-bottom: 1.25rem;
+        padding: 0.875rem 0.5rem 0 0.5rem;
+        border-top: 4px rgba(0, 0, 0, 0.05) solid;
         button {
-          flex-grow: 1;
+          color: var(--primary60);
+          font-weight: 600;
+        }
```

Comment:
No effect requested by @thibautsailly for hover ? Also you probably don't have to put these props in the `button` element

Reply:
I didn't see anything on the mockup. I also just asked him

Reply:
It's done for css

Reply:
Done

Reply:
Now I see this 
![Capture d’écran 2024-10-01 à 14 33 51](https://github.com/user-attachments/assets/6acba9a1-a2c7-4e35-a423-b64367db31a2)

From which mockup does it come from ?

Reply:
Nevermind